frontend/app/src/App.tsx

```diff
@@ -432,6 +439,17 @@ export class App extends PureComponent<Props, State> {
       },
     })
 
+    this.hostCommunicationMgr.sendMessageToHost({
+      type: "SCRIPT_RUN_STATE_CHANGED",
+      scriptRunState: this.state.scriptRunState,
+    })
```

Comment:
What's the reasoning for the Script Run state changed event here? Is it just for the test?

I offered an idea that might make more sense, but in any case a comment here would be helpful to relate the two concepts if they are related.

Reply:
Hm, this was just me following the prototype for this feature. Thinking through this, I don't think it's important so just moved it back to where it was in `componentDidMount`. @sfc-gh-pfinnigan do you happen to remember if this needs to happen each time a new `ConnectionManager` is constructed on the Snowsight side of things?